Update password docs

kef002 opened the pull request:
<!--  Thanks for sending a pull request!  Here are some tips for you -->

**What this PR does / why we need it**:

The PR is a small change in documentation to highlight the new mandatory update password feature 

**Special notes for your reviewer**:
 issue: #4840

**Checklist:**
- Confirm is the image is visible.

ispeakc0de commented on docs/kubelet-service-kill.md:
```diff
@@ -0,0 +1,196 @@
+  <tr>
+    <td> Generic </td>
+    <td> Kill Kubelet Service of Application Node </td>
+    <td> GKE, EKS </td>
```
Can we make them same as csv file?

ispeakc0de commented on docs/kubelet-service-kill.md:
```diff
@@ -0,0 +1,196 @@
+
+## Entry Criteria
+
+- Application Node is healthy before chaos injection
```
we are checking for the application status in pre/post chaos as well?

uditgaurav replied:
its application node

ispeakc0de replied:
This is the entry criteria right?
https://github.com/litmuschaos/litmus/blob/master/experiments/generic/kubelet_service_kill/kubelet_service_kill_ansible_logic.yml#L54

ispeakc0de commented on docs/kubelet-service-kill.md:
```diff
@@ -0,0 +1,196 @@
+## Details
+
+- This experiment Causes the application to become unreachable on account of node turning unschedulable (NotReady) due to kubelet service kill.
+- The kubelet service kill has been injected gracefully on a  node to make it unschedulable for a certain TOTAL_CHAOS_DURATION. The application node should be healthy once chaos is stopped and the services are reaccessable
```
The kubelet service has been stopped/killed on a node to make it unschedulable for a certain duration say `TOTAL_CHAOS_DURATION`. The application node should be healthy after the chaos injection and the services should be reaccessable

ispeakc0de commented on docs/kubelet-service-kill.md:
```diff
@@ -0,0 +1,196 @@
+  </tr>
+  <tr>
+    <td> RAMP_TIME </td>
+    <td> Period to wait before injection of chaos in sec </td>
```
Period to wait before & after injection of chaos in sec

ispeakc0de commented on docs/kubelet-service-kill.md:
```diff
@@ -0,0 +1,196 @@
+## Details
+
+- This experiment Causes the application to become unreachable on account of node turning unschedulable (NotReady) due to kubelet service kill.
+- The kubelet service has been stopped/killed on a node to make it unschedulable for a certain duration sai.e `TOTAL_CHAOS_DURATION`. The application node should be healthy after the chaos injection and the services should be reaccessable.
```
sai.e ?

ispeakc0de commented on docs/kubelet-service-kill.md:
```diff
@@ -0,0 +1,196 @@
+
+## Exit Criteria
+
+- Application pods will be healthy post chaos injection.
```
Application pods and the node should be healthy post chaos injection

ispeakc0de commented on docs/kubelet-service-kill.md:
```diff
@@ -0,0 +1,196 @@
+
+## Entry Criteria
+
+- Application pods are healthy on the respective Nodes before chaos injection.
```
Application pods should be healthy before chaos injection

rajdas98 commented on docs/kubelet-service-kill.md:
```diff
@@ -0,0 +1,196 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kubelet-service-kill
```
This service account name is not using in chaosengine spec

rajdas98 commented on docs/kubelet-service-kill.md:
```diff
@@ -0,0 +1,196 @@
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: nginx-chaos
```
name should be kubelet-chaos. Change it in other places

rajdas98 commented on docs/kubelet-service-kill.md:
```diff
@@ -0,0 +1,196 @@
+
+## Entry Criteria
+
+- Application pods should be healthy before chaos injection.
```
- Application pods should be healthy before chaos injection.
- Nodes should be healthy before chaos injection

rajdas98 commented on docs/kubelet-service-kill.md:
```diff
@@ -0,0 +1,196 @@
+
+## Exit Criteria
+
+- Application pods and the node should be healthy post chaos injection.
```
- Application pods should be healthy after chaos injection.
- Nodes should be healthy after chaos injection

kef002 committed:
Update reset-password.md

- I have added another mandatory method of updating the password.
- Users will be fully aware that they need to change the passwords if the login with the default passwords.
- This new feature is introduced in litmus 3.9.1 and above.

Signed-off-by: Charlotte Meme Kefeletswe <Charlotte.meme.kefeletswe@ericsson.com>

kef002 committed:
Added mandatory update password screenshot.

Signed-off-by: Charlotte Meme Kefeletswe <Charlotte.meme.kefeletswe@ericsson.com>

kef002 committed:
Update reset-password.md

- added the screenshot for update password step

Signed-off-by: Charlotte Meme Kefeletswe <Charlotte.meme.kefeletswe@ericsson.com>

kef002 committed:
Update reset-password.md

Signed-off-by: Charlotte Meme Kefeletswe <Charlotte.meme.kefeletswe@ericsson.com>

neelanjan00 commented on website/docs/user-guides/reset-password.md:
```diff
@@ -20,6 +20,13 @@ In the **Reset Password** modal type in the new password in the **New Password**
+
+## 3. Update password after installation
+
+After installing Chaos Litmus and entering the default passwords, users will be redirected to the update password page, which is mandatory. In the update password modal, enter your Old Password, New Password, and Re-enter New Password. Once done, click the Confirm button.
```
```suggestion
After installing LitmusChaos and authenticating using the default password in the UI, users will be mandatorily prompted to update the password. To update the password, enter the old (default) password, the new password, and re-enter the new password. Once done, select the **Confirm** button.
```

kef002 replied:
Thank you for the suggestion, I have made the changes accordingly.

kef002 committed:
Update reset-password.md

Edited wording as per the suggestions.

Signed-off-by: Charlotte Meme Kefeletswe <Charlotte.meme.kefeletswe@ericsson.com>

kef002 committed:
Fixed the picture file type to be png instead of PNG

Signed-off-by: Charlotte Meme Kefeletswe <Charlotte.meme.kefeletswe@ericsson.com>
